ntp-udp/src/raw_socket.rs

```diff
@@ -253,14 +251,14 @@ mod recv_message {
         };
 
         if mhdr.msg_flags & libc::MSG_TRUNC > 0 {
-            warn!(
+            tracing::info!(
```

Comment:
This loglevel seems still too high for me. This can be remotely triggered and doesn't neccessarily indicate problems on our end.

Reply:
Choice: debug